pkg/kv/kvserver/metrics.go

```diff
@@ -1221,6 +1221,37 @@ histogram.
 		Measurement: "Processing Time",
 		Unit:        metric.Unit_NANOSECONDS,
 	}
+	metaRaftCommandsProposed = metric.Metadata{
+		Name: "raft.commands.proposed",
+		Help: `Count of Raft commands proposed.
+
+This measurement is taken on the leaseholders, and includes both the original
+proposals and re-proposals. This approximates the number of commands submitted
+through Raft.`,
+		Measurement: "Commands",
+		Unit:        metric.Unit_COUNT,
+	}
+	metaRaftCommandsReproposed = metric.Metadata{
+		Name: "raft.commands.reproposed.same",
+		Help: `Count of Raft commands re-proposed without modification.
+
+This measurement is taken on the leaseholders, and counts retries of proposals
+that fail to be committed/applied within a certain amount of time, and are
+likely dropped. Such re-proposals make sure that all proposals are eventually
+committed to Raft and applied.`,
+		Measurement: "Commands",
+		Unit:        metric.Unit_COUNT,
+	}
+	metaRaftCommandsReproposedLAI = metric.Metadata{
+		Name: "raft.commands.reproposed.lai",
+		Help: `Count of Raft commands re-proposed with a later LAI.
+
+This measurement is taken on the leaseholders, and counts retries of proposals
```

Comment:
Maybe instead say what lai change mean for proposal? Like timeout in reproposed.unchanged.